.editorconfig

```
@@ -7,18 +7,31 @@ root = true
 # Default settings:
 # Use 4 spaces as indentation
 [*]
-indent_style = space
+charset = utf-8
```

Marking as request changes pending discussion of encoding.

It is my recommendation that the `charset` either not be listed under `[*]`, or be listed as `utf-8-bom` with exclusions for specific file types that define their own expected encoding (e.g. JSON).

```suggestion
charset = utf-8-bom
```

---

I'm not with the changes. This is in draft for a reason!